build-utils/cmake/modules/FindFbx.cmake

```diff
@@ -129,14 +126,51 @@ function(FindFbxLibrariesGeneric _FBX_ROOT_DIR _OUT_FBX_LIBRARIES _OUT_FBX_LIBRA
       endif()
     endif()
 
+    FindFbxVersion(${FBX_ROOT_DIR} PATH_VERSION)
+   
+    # 2019 SDK needs to link against bundled libxml and zlib on windows
+    if("x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC" AND "${FBX_VERSION_MAJOR}" STREQUAL "2019")
```

Comment:
Using cmake syntax VERSION_GREATER_EQUAL "2019.1" you could avoid the need of adding FBX_VERSION_MAJOR and make it forward compatible.